FAQ.md

```diff
@@ -26,9 +27,12 @@ Read [How does Dapr work with service meshes?](https://github.com/dapr/dapr/wiki
 
 Istio is not a programming model and does not focus on application level features such as state management, pub-sub, bindings etc. That is where Dapr comes in.
 
+## Performance Benchmarks
+The Dapr project is focused on performance due to the inherent discussion of Dapr being a sidecar to your application. This [performance benchmark video](https://youtu.be/4kV3SHs1j2k?t=783) discusses and demos the work that has been done so far. The performance benchmark data is planned to be published on a regular basis along with the tests that you can run in your own test environment.

```

Comment:
The tests are already published for several months now. It's the data for the tests we (maintainers) ran on a specific configuration that needs to be published.

Reply:
Ok. Looks good. Can you approved this based on your changes?

Reply:
I didn't make the changes.

Reply:
Ok. So it says exactly that "The performance benchmark data is planned to be published on a regular basis along with the tests that you can run in your own test environment." 
No changes needed

Reply:
Ok, so my thinking was that it might taken that the tests themselves should be published along with the test data, which is not the case.

I suggest to rephrase to say that only the test benchmarks will be published.

Reply:
How about this change
"The performance benchmark data is planned to be published on a regular basis. You can also run the perf tests in your own environment to get perf numbers."